OZON.ru.js

```diff
@@ -1,604 +1,103 @@
 {
 	"translatorID": "73be930f-5773-41b2-a7a1-37c0eeade92f",
 	"label": "OZON.ru",
-	"creator": "Victor Rybynok",
-	"target": "^https?://www\\.ozon\\.ru",
-	"minVersion": "2.1.9",
+	"creator": "Mikhail Akimov",
+	"target": "http://www.ozon.ru/context/detail/id/\\d{3,}",
+	"minVersion": "1.0",
 	"maxVersion": "",
 	"priority": 100,
 	"inRepository": true,
 	"translatorType": 4,
-	"browserSupport": "gcsibv",
-	"lastUpdated": "2014-04-03 18:47:00"
+	"browserSupport": "gcsi",
+	"lastUpdated": "2015-09-22 10:03:30"
 }
 
-/**
-	OZON.ru translator. Only supports "Book" "Item Type".
-	Copyright (c) 2012 Victor Rybynok, v.rybynok@gmail.com
+/*
+	ACM Digital Library Translator
+	Copyright (C) 2015 Mikhail Akimov
 
-	Description
-	===========
-
-	Zotero translator for OZON.ru on-line store.
-
-	Comments
-	========
-
-	This version of OZON.ru translator only supports "Book" "Item Type"
-	for "Книги" and "Цифровые книги" ozon.ru site sections.
-
-	Change log
-	==========
-
-	2013-05-27
-
-	* New page layout parsing.
-
-	2012-08-11
-
-	* Corrected detail description parsing to accommodate the new book
-	  description pages template.
-	* Corrected "edition number" recognition.
-	* Introduced change log and to do comment sections.
-	* Few minor issue fixes.
-
-	2012-04-02
-
-	* Initial release.
-
-	TODO
-	====
-
-	* Develop more universal algorithm for "edition number" recognition.
-	  Currently it only recognises up to 10 editions in Russian verbal
-	  representation.
-
-	Legal notice
-	============
-
-	This program is free software: you can redistribute it and/or
-	modify it under the terms of the GNU Affero General Public License
-	as published by the Free Software Foundation, either version 3 of
-	the License, or (at your option) any later version.
+	This program is free software: you can redistribute it and/or modify
+	it under the terms of the GNU General Public License as published by
+	the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
 
 	This program is distributed in the hope that it will be useful,
 	but WITHOUT ANY WARRANTY; without even the implied warranty of
-	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
-	Affero General Public License for more details.
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+	GNU General Public License for more details.
 
-	You should have received a copy of the GNU Affero General Public
-	License along with this program.  If not, see
-	<http://www.gnu.org/licenses/>.
+	You should have received a copy of the GNU General Public License
+	along with this program. If not, see <http://www.gnu.org/licenses/>.
```

Comment:
Note there are more non AGPL licences in translators: <a href="https://github.com/zotero/translators/search?utf8=%E2%9C%93&q=%22copy+of+the+GNU+General+Public+License%22&type=Code">111 results</a> . I guess it would be preferable if we manage to change them all?